clang/lib/DPCT/MapNames.cpp

```diff
@@ -364,6 +364,11 @@ void MapNames::setExplicitNamespaceMap() {
       {"cusparseHandle_t", std::make_shared<TypeNameRule>(
                                getDpctNamespace() + "sparse::descriptor_ptr")},
       {"cudaMemoryAdvise", std::make_shared<TypeNameRule>("int")},
+      {"cudaStreamCaptureStatus",
```

Comment:
When the extension graph isn't enabled, maybe we should emit a warning instead of migrating to `int` here.

Reply:
Done.